crates/kitsune_p2p/dht/src/arq/arq_set.rs

```diff
@@ -305,6 +304,95 @@ mod tests {
         );
     }
 
+    #[test]
+    fn arq_set_is_union() {
+        let dim = SpaceDimension::standard();
+        let strat = ArqStrat::default();
+
+        let start_a = 10_000;
+        let len_a = 30_000;
+        let arq_a =
+            Arq::from_start_and_half_len_approximate(dim, &strat, start_a.into(), len_a / 2);
+
+        let start_b = 10_000_000;
+        let len_b = 20_000;
```

Comment:
These are extremely small arcs. The quantum size is 2^12 = 4096. For perspective, the quantum size was selected to support sharded networks with millions of connected nodes ([justification](https://github.com/holochain/holochain/blob/c5480bea43b0810b395a3fdd2e24653795aa76b2/crates/kitsune_p2p/dht/src/spacetime/topology.rs#L162-L176)). So while it's good to test at arbitrary scales, this is a scale we won't be seeing any time soon, or perhaps ever.